app/controllers/ems_common.rb

```diff
@@ -694,6 +700,9 @@ def get_form_vars
         @edit[:new][:port] = @ems.port ? @ems.port : ManageIQ::Providers::OpenshiftEnterprise::ContainerManager::DEFAULT_PORT
       elsif params[:server_emstype] == ManageIQ::Providers::AtomicEnterprise::ContainerManager.ems_type
         @edit[:new][:port] = @ems.port ? @ems.port : ManageIQ::Providers::AtomicEnterprise::ContainerManager::DEFAULT_PORT
+      elsif params[:server_emstype] == "vmware_cloud"
```

Comment:
@gberginc should not need to set anything in ``` edit[:new] ``` angular form does not go thru get_form_vars method and set values in ``` @edit[:new] ```